bundles/binding/org.openhab.binding.temperlan/OSGI-INF/binding.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2015, openHAB.org and others.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" name="org.openhab.binding.temperlan.binding">
+	<implementation class="org.openhab.binding.temperlan.internal.TemperlanBinding" />
+
+	<service>
+		<provide interface="org.osgi.service.event.EventHandler" />
+		<provide interface="org.osgi.service.cm.ManagedService" />

```

Comment:
using the ManagedService is deprecated. Please have a look into the binding archetype for details about getting the configuration injected